boards/espressif/esp32c6_devkitc/doc/index.rst

```diff
@@ -0,0 +1,264 @@
+.. _esp32c6_devkitc:
+
+ESP32-C6-DevKitC-1
+##################
+
+Overview
+********
+
+ESP32-C6 is Espressif’s first Wi-Fi 6 SoC integrating 2.4 GHz Wi-Fi 6, Bluetooth 5.3 (LE) and the
```

Comment:
typo: `Espressif's`

Reply:
Is that a typo? The possessive form "Espressif’s" means that ESP32-C6 is a product of Espressif